x-pack/plugins/enterprise_search/server/lib/enterprise_search_config_api.ts

```diff
@@ -67,44 +67,60 @@ export const callEnterpriseSearchConfigAPI = async ({
       publicUrl: stripTrailingSlash(data?.settings?.external_url),
       readOnlyMode: !!data?.settings?.read_only_mode,
       ilmEnabled: !!data?.settings?.ilm_enabled,
+      isFederatedAuth: !!data?.settings?.is_federated_auth, // i.e., not standard auth
       configuredLimits: {
-        maxDocumentByteSize: data?.settings?.configured_limits?.max_document_byte_size,
-        maxEnginesPerMetaEngine: data?.settings?.configured_limits?.max_engines_per_meta_engine,
+        appSearch: {
+          engine: {
+            maxDocumentByteSize:
+              data?.settings?.configured_limits?.app_search?.engine?.document_size_in_bytes,
+            maxEnginesPerMetaEngine:
+              data?.settings?.configured_limits?.app_search?.engine?.source_engines_per_meta_engine,
+          },
+        },
+        workplaceSearch: {
+          customApiSource: {
+            maxDocumentByteSize:
+              data?.settings?.configured_limits?.workplace_search?.custom_api_source
+                ?.document_size_in_bytes,
+            totalFields:
+              data?.settings?.configured_limits?.workplace_search?.custom_api_source?.total_fields,
+          },
+        },
```

Comment:
@scottybollinger Not sure if Workplace Search actually uses this configured limit data anywhere in the UI but I figured I might as well add it!